join_request/xlc.yml

```
@@ -0,0 +1,22 @@
+collective: Potoc
+member:
+  name: Bryan
+  address: 14DsLzVyTUTDMm2eP3czwPbH53KgqnQRp3CJJZS9GR7yxGDP
+  github: xlc
```

please fill your matrix handle as per https://github.com/super-collective/collective-cli/pull/4#discussion_r1586380064
```suggestion
  address: 14DsLzVyTUTDMm2eP3czwPbH53KgqnQRp3CJJZS9GR7yxGDP
  matrix:
  github: xlc
```